src/main/java/org/bigredbands/mb/controllers/PDFGenerator.java

```diff
@@ -116,8 +116,8 @@ public void createPDF(DrillInfo drillInfo, File file) throws IOException {
 
         // Drill image dimensions
         // Units are in pixels
-        float imageWidth = drillWidth;
-        float imageHeight = drillHeight;
+        float imageWidth = drillWidth * (300f / 72f);
```

Comment:
Let's add a comment here about why this multiplication is happening, and why the values are set to 300 and 72 respectively.